lib/src/facebook_login.dart

```diff
@@ -26,9 +28,13 @@ class FacebookLogin {
 
   /// If `true` all requests and results will be printed in console.
   final bool debug;
+	/// Initialize with Facebook App ID
+  final String appId;
 
-  FacebookLogin({this.debug = false}) {
+  FacebookLogin({this.debug = false, required this.appId}) {
```

Comment:
Shouldn't be required. It's completely optional and all existed projects should be working without any changes.